packages/testing-helpers/src/scopedElementsWrapper.js

```diff
@@ -0,0 +1,64 @@
+import { ScopedElementsMixin } from '@open-wc/scoped-elements';
+import { html, LitElement, TemplateResult } from 'lit-element';
+import { nothing } from 'lit-html';
+import { isIterable } from './lib.js';
+
+const transform = template => {
+  if (isIterable(template)) {
+    return [...template].map(v => transform(v));
+  }
+
+  if (template instanceof TemplateResult) {
+    return html(template.strings, ...template.values);
+  }
+
+  return template;
+};
+
+class ScopedElementsClass extends ScopedElementsMixin(LitElement) {
+  static get properties() {
+    return {
+      scopedElements: { type: Object },
+      template: { type: Object },
+    };
+  }
+
+  constructor() {
+    super();
+
+    this.scopedElements = {};
+    this.template = nothing;
+  }
+
+  async firstUpdated() {
+    // @ts-ignore
+    await super.firstUpdated();
+
+    Object.keys(this.scopedElements).forEach(key =>
+      this.defineScopedElement(key, this.scopedElements[key]),
+    );
+  }
+
+  render() {
+    return transform(this.template);
+  }
+}
+
+// @ts-ignore
```

Comment:
why?

Reply:
Same as before but in this case the is not inferring the `CustomElementConstructor`.

![Screenshot 2020-04-24 at 15 43 19](https://user-images.githubusercontent.com/329771/80219074-53cb0900-8642-11ea-81b7-c756bbf9b803.png)

Reply:
see above